ocs_ci/templates/mcg/stress-cli-sts.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  namespace: openshift-storage
+  name: nb-stress-cli
+spec:
+  selector:
+    matchLabels:
+      app: stress-cli
+  replicas: 2
+  updateStrategy:
+    type: RollingUpdate
+  podManagementPolicy: OrderedReady
+  volumeClaimTemplates: []
+  template:
+    metadata:
+      labels:
+        app: stress-cli
+    spec:
+      volumes:
+        - name: service-ca
+          configMap:
+            name: stress-cli-service-ca
+      containers:
+        - name: stress-cli
+          image: quay.io/ocsci/nb-stress-cli:latest
```

Comment:
Please add some documentation about what this image is for. I'm not sure if it's possible to add it in this YAML, but at least elaborate about it in the description of the PR.